esp/esp/resources/models.py

```diff
@@ -169,6 +169,7 @@ class Resource(models.Model):
     is_unique = models.BooleanField(default=False)
     user = AjaxForeignKey(ESPUser, null=True, blank=True)
     event = models.ForeignKey(Event)
+    attribute_value = models.TextField(default="")
```

Comment:
This [needs to have blank=True](https://docs.djangoproject.com/en/1.8/ref/models/fields/#blank) so people will be able to edit it from the admin panel or other forms. (Then you'll want to delete and regenerate the migration.)

It's also not clear from the code what it's for. I don't have bright ideas for better names, so maybe add a comment?